### DISCUSSION TITLE
"No Internet Access" Even Though It Still Works

### DISCUSSION THREAD

**Discussion by Underplayer97** [DNSCrypt/dnscrypt-proxy#2325] [Q&A] [answered]
I have been using DNSCrypt to bypass a school internet filter but when I connect to the access point, it says there is no internet even though I can still be able to go through. This has messed up many things using the settings in Windows saying there is no internet even if it still works. Is there any way to make it show that there is internet?

**Comment by jedisct1** [comment, marked as answer]
Not sure what "it" refers to, but if this is the network status indicator, maybe this page in the documentation helps? https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Windows-NCSI

**Reply by Underplayer97** [reply]
Sorry, I meant "it" as the internet connection but I looked at the documentation and tried the things on there and the network indicator still shows as "No Internet access"

**Reply by lifenjoiner** [reply]
You should start from https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-Windows#troubleshooting, change the system settings and reboot.

And according to [NCSI active probes and the network status](https://learn.microsoft.com/en-us/troubleshoot/windows-client/networking/internet-explorer-edge-open-connect-corporate-public-network#ncsi-active-probes-and-the-network-status-alert), the HTTP downloading failure also counts. That is out of a DNS server's ability.

**Reply by Underplayer97** [reply]
It took a minute for me to get back onto the network but I was successfully able to get the NCSI Indicator to show as "Internet Access"
Thanks for finally getting this issue fixed! This issue persisted for far too long for me and its nice to finally not have Windows yell at me saying there is no internet when it actually works.
